make/modules/java.desktop/lib/Awt2dLibraries.gmk

```diff
@@ -132,6 +132,7 @@ $(eval $(call SetupJdkLibrary, BUILD_LIBAWT, \
     EXCLUDE_FILES := $(LIBAWT_EXFILES), \
     OPTIMIZATION := HIGHEST, \
     CFLAGS := $(CFLAGS_JDKLIB) $(LIBAWT_CFLAGS), \
+    CXXFLAGS := $(CXXFLAGS_JDKLIB) $(LIBAWT_CFLAGS), \
```

Comment:
`LIBAWT_CFLAGS` applies to both C and C++, right? It contains some common flags; it follows the pattern with other libs below.

Reply:
That's correct. `LIBAWT_CFLAGS contains a bunch of `-D` flags, which are common for C and C++, and `-EHsc` flag, which is probably C++ only, but the compiler doesn't complain.

`jabswitch` below is using some C++-specific flags, but it only compiles one C++ file and no C files. I suppose we could simplify that makefile at some point. All other flags are common between C and C++.

Reply:
Thank for clarification.